ui/temboardui/templates/base.html

```diff
@@ -21,54 +21,55 @@
     <script type="module" src="{{ vitejs.url_for('temboard.js') }}"></script>
     <!-- Navigation -->
     <div class="navbar bg-primary navbar-static-top navbar-expand sticky-top p-0" data-bs-theme="dark">
-      <div class="collapse navbar-collapse">
-        {% if 'instance' in globals() %}
-        <a class="navbar-brand ps-0 ms-3 me-0" href id="menu-collapse">
-          <i class="fa fa-bars"></i>
-        </a>
+      {% if 'instance' in globals() %}
+      <a class="navbar-text fs-6 ms-3 text-white" href id="menu-collapse">
+        <i class="fa fa-bars"></i>
+      </a>
+      {% end %}
+      <a class="navbar-brand fs-5 p-0 ms-3" href="/home">temBoard</a>
+      {% if 'instance' in globals() %}
+      {% set d = instance.discover or {} %}
+      <span class="navbar-text p-0 fs-5 text-white text-truncate">
+        /
+        {% if instance.groups %}
+        <span>{{instance.groups[0].group_name}}</span>
```

Comment:
I know that in a near future an instance will belong to one group only. But it's not currently the case, and I think choosing the first one is quite arbitrary.

Reply:
Oui.